scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -71,14 +96,14 @@ task MergeVCFs {
 task Filter {
   File gatk4_jar
   File unfiltered_calls
-  String output_vcf_name
+  String output_prepend
 
   command {
-  	java -Xmx4g -jar ${gatk4_jar} FilterMutectCalls -V ${unfiltered_calls} -O ${output_vcf_name}.vcf
+  	java -Xmx4g -jar ${gatk4_jar} FilterMutectCalls -V ${unfiltered_calls} -O ${output_prepend}-filtered.vcf
   }
 
   output {
-    File output_vcf = "${output_vcf_name}.vcf"
+    File output_vcf = "${output_prepend}-filtered.vcf"
```

Comment:
I understand you to be referring to the variable name `output_vcf` as opposed to its value `"${output_prepend}-filtered.vcf"`.  Let me know if I'm mixed up.